Library/Homebrew/keg_relocate.rb

```diff
@@ -45,6 +48,17 @@ def replace_text(text)
       end
       any_changed
     end
+
+    sig { params(value: T.any(String, Regexp)).returns(Regexp) }
+    def self.path_regex(value)
```

Comment:
```suggestion
    def self.path_to_regex(path)
```

Suggestion here, not binding at all and feel free to disagree.